.github/workflows/upgrade_test_constraints.yml

```diff
@@ -23,6 +16,7 @@ concurrency:
 jobs:
   upgrade:
     name: Upgrade & Open Pull Request
+    if: (github.event.issue.pull_request != '' && contains(github.event.comment.body, '@napari-bot constraints update')) || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule'
```

Comment:
```suggestion
    if: (github.event.issue.pull_request != '' && contains(github.event.comment.body, '@napari-bot update constraints')) || github.event_name == 'workflow_dispatch' || github.event_name == 'schedule'
```

Reply:
I've updated the language to "update constraints", which sounds more like a command, which is the common format for interacting with bots